### ISSUE TITLE
HJR TK499

### ISSUE THREAD

**Issue by ajsb85** [zlgopen/awtk#575]
Dear @xianjimli 

I bought this development board.
I will use for a IoT project.
Do you think this board is a good candidate for AWTK?

![image](https://user-images.githubusercontent.com/663460/125030029-c4651680-e08a-11eb-9613-4428353e81b9.png)

Development files:
https://whycan.com/t_5861.html

**Comment by ajsb85** [comment]
I bought all these components:
https://es.aliexpress.com/item/4001095715528.html?spm=a2g0s.9042311.0.0.274263c0kMKnPX

**Comment by xianjimli** [comment]
Yes, it is. The author has ported AWTK (reference: https://www.amobbs.com/thread-5720658-1-1.html), but I don't know if this board supports AWTK, but the porting is not difficult.

**Comment by ajsb85** [comment]
Do you agrees if we sponsor this board to the AWTK?
Can you provide a shipping address?

**Comment by xianjimli** [comment]
Thank you. There is no need. In China, it is very convenient to buy various development boards. If you have problems with porting, just contact me.
